### PR TITLE
testing locally

### PR DESCRIPTION
**Pull request by acarnagey**

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -29,8 +29,8 @@ jobs:
       - name: Test
         run: npm run test
 
-      - name: Deploy test report
-        uses: peaceiris/actions-gh-pages@v3
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./docs
+      # - name: Deploy test report
```
Comment (gjgd) [inline .github/workflows/cd.yml:32]: would prefer to remove or to give some context as to why this step is commented out

**Review by gjgd — APPROVED** [review]

File: .github/workflows/cd.yml (hunk 1 above)
Comment (OR13) [reply .github/workflows/cd.yml:32]: because we don't need to build a report for the private fork we are using to test the vdp.

**Review by OR13 — CHANGES_REQUESTED** [review]
1. create a bug ticket in the VDP for "VC-HTTP-API tests are passing but should not be", with description noting the changes you made here.
2. revert all changes made here to "get the tests to pass" (preserving those changes on the ticket above)
3. Root cause the 503 error, and open a ticket in the VDP to track it.


as long as tickets for 1 and 3 exist in the VDP backlog, and this PR shows us passing / failing tests as they exists in the public repo, this PR can be merged.

**Comment by acarnagey** [comment]
Seems to be running out of memory when handling that many requests at once https://transmute.atlassian.net/browse/VDP-2168

**Pull request merged**